src/utils/diff/diff.ts

```diff
@@ -1,8 +1,14 @@
 import { execSync } from 'node:child_process'
 import parse from 'parse-diff'
+import { readFileSync } from 'fs'
 
 export const executeDiff = (diffCommand: string): parse.File[] => {
-    const result = execSync(`git diff ${diffCommand}`)
-
-    return parse(result.toString())
+    let result
+    try {
+        execSync(`git diff ${diffCommand} > diff.txt`, { stdio: 'ignore' })
+        result = parse(readFileSync('diff.txt', 'utf8'))
```

Comment:
you can also just return here you don't need the result variable

Reply:
didnt realize finally gets called after a return. Ive updated it